### ISSUE TITLE
Students sorted by their firstname

### ISSUE THREAD

**Issue by tanchon** [chamilo/chamilo-lms#2299]
### Current behavior / Resultado actual / Résultat actuel
Today i work with chamilo (adding users, class creation, announcements...). 

After a long time of work, I think that something is not very "ergonomic" : when we want to add users in a class (from the admin panel, not using csv files), users (students, teachers...) are sorted by their firstname when we want to add them using the list. 

It is the same thing with announcements.

### Expected behavior / Resultado esperado / Résultat attendu
I think it would be more practical to sort users by their lastname.

### Steps to reproduce / Pasos para reproducir / Étapes pour reproduire
Adding users in classes (in admin panel) ; sending an announcement (form a course).

### Chamilo Version / Versión de Chamilo / Version de Chamilo
1.11.4

**Comment by jmontoyaa** [comment]
The order depends in the language you're using chamilo. That's the default in English. If you want to change it you can edit this file and add something like:

app/config/configuration.php

```
$_configuration['name_order_conventions'] = array(
    'french' => array('format' => 'title last_name first_name', 'sort_by' => 'last_name')
    'english => array('format' => 'title last_name first_name', 'sort_by' => 'last_name')
);
```

**Comment by tanchon** [comment]
Thanks a lot jmontoyaa ! It works :-) 

 It's a very useful tip ! It is far more ergonomic now.

 I read this configuration.php and have done some modifications, but i didn't pay attention to this parameter.

Moreover, it solved #2301  : when activate this option, there is no more issue as i mentioned here (but if i desactivate the option, the issue #2301  is "reactivated".
